### PR TITLE
fix: updates nvidia platform services installation

### PR DESCRIPTION
**Pull request by tillt**
## Jira Ticket

Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-81538

## Description of changes being made
Fixes an outdated description of the setup procedure for enabling Nvidia platform services on Kommander 2.1.  Should not wait for a new release, if possible.

## Checklist

- [x] Test all commands and procedures, if applicable.
- [x] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](./CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/gpu/kommander-config/index.md [hunk 1]
```diff
@@ -36,13 +36,31 @@ Before you begin, you must:
 
 ## Enable Nvidia Platform Service on Kommander
 
-To enable Nvidia GPU support on Kommander, perform the following step:
+To enable Nvidia GPU support when installing Kommander, perform the following steps:
+
+1. Kommander v2.1 installs with a dedicated CLI.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/gpu/kommander-config/index.md:41]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Kommander v2.1 installs with a dedicated CLI.</p>
```

File: pages/dkp/kommander/2.1/gpu/kommander-config/index.md [hunk 2]
```diff
@@ -36,13 +36,31 @@ Before you begin, you must:
...
+
+1. Append to the apps section in install.yaml with the following to enable Nvidia platform services.
+
+
+    ```yaml
+    apps:
+      nvidia: null
+    ```
+
+1. To install Kommander using the above configuration file, enter the following command:
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/gpu/kommander-config/index.md:57]:
```suggestion
1. Install Kommander, using the configuration file you created in the previous step, with this command:
```

File: pages/dkp/kommander/2.1/gpu/kommander-config/index.md [hunk 3]
```diff
@@ -36,13 +36,31 @@ Before you begin, you must:
...
+1. Kommander v2.1 installs with a dedicated CLI.
+
+1. Create an installation configuration file:
+
+    ```bash
+    kommander install --init > install.yaml
+    ```
+
+1. Append to the apps section in install.yaml with the following to enable Nvidia platform services.
+
```
Comment (Colin1841) [inline pages/dkp/kommander/2.1/gpu/kommander-config/index.md:50]:
```suggestion
```

**Review by Colin1841 — APPROVED** [review]

**Comment by cmastr** [comment]
@tillt this will need to go into the 2.2 branch as well please.

File: pages/dkp/kommander/2.1/gpu/kommander-config/index.md [hunk 4]
```diff
@@ -36,13 +36,30 @@ Before you begin, you must:
 
 ## Enable Nvidia Platform Service on Kommander
 
-To enable Nvidia GPU support on Kommander, perform the following step:
+To enable Nvidia GPU support when installing Kommander, perform the following steps:
+
+<p class="message--note"><strong>NOTE: </strong>Kommander v2.1 installs with a dedicated CLI.</p>
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/gpu/kommander-config/index.md:41]:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Kommander installs with a dedicated CLI.</p>
```

File: pages/dkp/kommander/2.1/gpu/kommander-config/index.md [hunk 5]
```diff
@@ -36,13 +36,30 @@ Before you begin, you must:
...
+    ```
+
+1. Append to the apps section in install.yaml with the following to enable Nvidia platform services.
+
+    ```yaml
+    apps:
+      nvidia: null
+    ```
+
+1. Install Kommander, using the configuration file you created in the previous step, with this command:
```
Comment (cmastr) [inline pages/dkp/kommander/2.1/gpu/kommander-config/index.md:56]:
```suggestion
1. Install Kommander, using the configuration file you created:
```

**Comment by tillt** [comment]
https://github.com/mesosphere/dcos-docs-site/pull/3944 is the 2.2 version

File: pages/dkp/kommander/2.1/gpu/kommander-config/index.md [hunk 6]
```diff
@@ -36,13 +36,30 @@ Before you begin, you must:
...
+
+<p class="message--note"><strong>NOTE: </strong>Kommander installs with a dedicated CLI.</p>
+
+1. Create an installation configuration file:
+
+    ```bash
+    kommander install --init > install.yaml
+    ```
+
+1. Append to the apps section in install.yaml with the following to enable Nvidia platform services.
```
Comment (msdolbey) [inline pages/dkp/kommander/2.1/gpu/kommander-config/index.md:49]:
```suggestion
1. Append the following to the apps section in the install.yaml file to enable Nvidia platform services.
```

**Review by msdolbey — APPROVED** [review]
Made a small nit, but looks good.
👍

**Comment by tillt** [comment]
@msdolbey that landed already - we now need to fix 
- master
- my second PR against 2.2

**Pull request merged**